src/main/scala/jigg/util/JSONUtil.scala

```diff
@@ -18,18 +18,55 @@ object JSONUtil {
 
   private def toJSONFromNode(node: Node): String = {
     val sb = new StringBuilder
-
+    val escapedsb = new StringBuilder
+    val returnsb = new StringBuilder
     sb.append('{')
     sb.append(List("\".tag\":\"",node.label,"\",").mkString)
     sb.append("\".child\":")
     sb.append("[")
-    
     sb.append(serializing(node))
-
     sb.append("]")
     sb.append("}")
+    val returnString = pretty(render(parse(escapeString(sb.toString,escapedsb).toString)))
+    returnString
+  }
+
+  /**
+    * Append escaped string as 's'.
+    * This method is borrowed from xml.Utility.escape
+    */
+  final def escapeString(text: String, s: StringBuilder): StringBuilder = {
```

Comment:
I found `replace` method in String (e.g., `str.replace("\\", "\\\\")`). Is this sufficient for escape/unescaping?